docs/installation.md

```diff
@@ -192,15 +192,9 @@ Add the following job in your `azure-pipelines.yaml` file
       # Run MegaLinter
       - script: |
           docker run -v $(System.DefaultWorkingDirectory):/tmp/lint \
-            -e GIT_AUTHORIZATION_BEARER=$(System.AccessToken) \
-            -e CI=true \
```

Comment:
**CI** variable is used within MegaLinter so should be kept in the docker run arguments:)